MathLib/LinAlg/PETSc/PETScVector.h

```diff
@@ -154,59 +156,41 @@ class PETScVector
             VecSetValues(_v, e_idxs.size(), &e_idxs[0], &sub_vec[0], INSERT_VALUES);
         }
 
-        //! Get several entries
-        std::vector<double> get(std::vector<IndexType> const& indices) const
-        {
-            std::vector<double> local_x(indices.size());
+        // TODO preliminary
```

Comment:
Is this comment needed. If yes could you please explain a little bit more in detail what have to be done?